context.go

```diff
@@ -282,7 +282,10 @@ func (c *context) RealIP() string {
 	if ip := c.request.Header.Get(HeaderXForwardedFor); ip != "" {
 		i := strings.IndexAny(ip, ",")
 		if i > 0 {
-			return strings.TrimSpace(ip[:i])
+			xffip := ip[:i]
+			xffip = strings.ReplaceAll(xffip, "[", "")
+			xffip = strings.ReplaceAll(xffip, "]", "")
+			return xffip
```

Comment:
I think IP returned at line 293 `if ip := c.request.Header.Get(HeaderXRealIP); ip != "" {` needs similar treatment.

and maybe instead of `strings.ReplaceAll` we could check if first char is `[` and last is `]` and then just cut them.